app/services/onlinetesting/Phase3TestService.scala

```diff
@@ -267,6 +270,8 @@ trait Phase3TestService extends OnlineTestService with Phase3TestConcern {
       for {
         testGroup <- phase3TestRepo.getTestGroupByToken(launchpadInviteId)
         _ <- phase3TestRepo.updateProgressStatus(testGroup.applicationId, ProgressStatuses.PHASE3_TESTS_RESULTS_RECEIVED)
+        // Launchpad only: If results have been sent for a user, unexpire them
+        _ <- removeExpiryStatus(testGroup.applicationId)
```

Comment:
Should removeExpiryStatus be done before the progress status change, or we may end up with PHASE3_TESTS_RESULTS_RECEIVED but expired candidates if Mongo fails?